.github/workflows/release.yml

```diff
@@ -35,3 +35,59 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+      - name: Publish updates to Storybook
+        if: steps.changesets.outputs.published == 'true'
+        uses: actions/checkout@v3
+
+      - name: Use Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: "16"
+          cache: "yarn"
+
+      - name: Install dependencies
+        run: yarn
```

Comment:
Just being curious, how does the release workflow work? (i see there are 3 steps that repeat this setup - install dependencies step... given it's the same repo, how is this not redundant? does something get clear after each step?)
sorry in advance if it is a dumb question, i have no clue of workflows

Reply:
These steps are redundant since they are in the same job.

Reply:
Oh yes good spot, will remove